.travis.yml

```diff
@@ -14,8 +14,8 @@ env:
   global:
     - DOC_URL=https://developer.shotgunsoftware.com
     - DOC_PATH=""
-    - S3_BUCKET=doctest3
-    - S3_WEB_URL=http://doctest3.s3-website.eu-west-2.amazonaws.com
+    - S3_BUCKET=sg-devdocs
```

Comment:
this is weird - is something wrong with the branch? Does it need to have master merged in? 
This change is already in master:
https://github.com/shotgunsoftware/developer.shotgunsoftware.com/blob/master/.travis.yml#L17

Reply:
I'll have to check.  I merged master in locally, but last time this happened I had to rebase (which adds a ton of commits to the history since I don't have permission to rewrite history on the remote).  I'll need to update the tk-doc-generator submodule once the PR is merged there (and this branch is marked as conflicted now) so I'll have to figure it out before merging.